### ISSUE TITLE
Is python 3.x supported on windows as well?

### ISSUE THREAD

**Issue by Coderx7** [Microsoft/caffe#94]
It is supported on linux I guess, but Is it also supported on windows? 
Thanks in advance

**Comment by zer0n** [comment]
Have you tried it?

**Comment by Coderx7** [comment]
yep and failed, I'm no sure if I did something wrong, or its not supported for some reason, thats why I asked the question after I searched . I couldnt find any one who successfully used python3 on widnows.

**Comment by RoyiAvital** [comment]
I'm looking for Python 3.5 (Using Anaconda) support as well.

Moreover, it seems the MATLAB interface is buggy and not working.
I have cases where the Command Line works yet the MATLAB interface crashes.
